src/Features/CSharp/Portable/CodeFixes/HideBase/HideBaseCodeFixProvider.RemoveNewKeywordAction.cs

```diff
@@ -0,0 +1,79 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CSharp.Extensions;
+using Microsoft.CodeAnalysis.CSharp.LanguageServices;
+
+namespace Microsoft.CodeAnalysis.CSharp.CodeFixes.HideBase
+{
+    internal partial class HideBaseCodeFixProvider
+    {
+        private class RemoveNewKeywordAction : CodeActions.CodeAction
```

Comment:
instead of this, it can just be a DocumetnChangeAction, and just provide an appropriate lambda to do the fixup.  This is what the majority of other fixers do.

Reply:
Followed this approach with `MyCodeAction` (looked through other fixers and found it there) in new separate `RemoveNewModifierCodeFixProvider`.